tests/components/minecraft_server/const.py

```diff
@@ -15,9 +17,9 @@
         "online": 3,
         "max": 10,
         "sample": [
-            {"name": "Player 1", "id": "1"},
-            {"name": "Player 2", "id": "2"},
-            {"name": "Player 3", "id": "3"},
+            {"name": "Dummy Player 1", "id": "1"},
+            {"name": "Dummy Player 2", "id": "2"},
+            {"name": "Dummy Player 3", "id": "3"},
```

Comment:
Why is this change needed?

Reply:
It is not needed for this PR. I'll revert it :+1: